datamonster_api/tests/lib/test_company.py

```diff
@@ -10,6 +10,11 @@ def _assert_object_matches_company(company, company_obj):
     assert company_obj["uri"] == company.uri
 
 
+def test_equality(company, other_company):
+    assert company == company
```

Comment:
When would this not be true?

Reply:
if someone makes a code change to break the equality logic (or the conftest itself)

Reply:
sure